src/Symfony/Component/Yaml/Command/LintCommand.php

```diff
@@ -30,6 +30,16 @@ class LintCommand extends Command
     private $parser;
     private $format;
     private $displayCorrectFiles;
+    private $directoryIteratorProvider;
+    private $isReadableProvider;
+
+    public function __construct($name = null, $directoryIteratorProvider = null, $isReadableProvider = null)
```

Comment:
Couldn't `$directoryIteratorProvider` and `$isReadableProvider` be type-hinted as `callable`?